Assets/LeapC/Finger.cs

```diff
@@ -356,9 +372,11 @@ public override string ToString ()
      * @returns The invalid Finger instance.
      * @since 1.0
      */
-        public static Finger Invalid {
```

Comment:
We actually deprecated the `Invalid` concept at this layer in the new API, you should be able to just remove all of this code once we update the wrappers here.